Documentation/v2/admin_guide.md

```diff
@@ -165,13 +165,16 @@ $ ssh 10.0.1.11
 ```
 
 ```sh
-$ kill `pgrep etcd`
+$ systemctl stop etcd2
```

Comment:
The problem is that docs on administering etcd that appear on coreos.com _don't work on CoreOS_.